### PR TITLE
ISM API

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: content/en/docs/setup/customizing/opensearch.md [hunk 1]
```diff
@@ -123,3 +125,34 @@ Containers:
     Requests:
       memory:   1500M
 ```
+
+## Configure Index State Management Policies
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/opensearch.md:129]: Configure Index State Management Policies -> Configure Index State Management policies

File: content/en/docs/setup/customizing/opensearch.md [hunk 2]
```diff
@@ -123,3 +125,34 @@ Containers:
...
+
+[Index State Management](https://opensearch.org/docs/1.3/im-plugin/ism/index/) policies configure OpenSearch to manage the data in your indices. 
+Policies can be used to automatically rollover and prune old data, preventing your OpenSearch
+cluster from running out of disk space.
+
+The following policy example configures OpenSearch to manage indices matching the pattern `my-app-*`. The data in these indices will be
+automatically pruned every 14 days, and will be rolled over if an index meets at least one of the following criteria:
+- Is 3 or more days old.
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/opensearch.md:137]: 3 -> three (guidance is: Spell out numbers zero through nine. Use numerals for 10 and higher. This holds unless you are specifying a data point, in which case, the number would be in code font, for example `3`)

**Review by rosemarymarano — COMMENTED** [review]
A few edits

File: content/en/docs/setup/customizing/opensearch.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ Verrazzano supports two cluster topologies for an OpenSearch cluster:
 [Installation Profiles](/docs/setup/install/profiles/) describes the default OpenSearch cluster
 configurations provided by Verrazzano.  
 
+## Configure Cluster Topology
+
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply content/en/docs/setup/customizing/opensearch.md:19]: Sure, I'm ok with that. The challenge is to make sure that they are kept identical.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 4]
```diff
@@ -250,6 +251,20 @@ spec:
 | `nodes.data.requests.memory` | [NameValue](#name-value) |  string  | The data node memory request amount expressed as a [Quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/#Quantity). |  No |
 | `nodes.data.requests.storage` | [NameValue](#name-value) |  string  | The data storage request amount expressed as a [Quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/#Quantity). |  No |
 
+#### OpenSearch Index Management Policies
```
Comment (kminder) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:254]: These aren't rendering in the file view so I wonder if there is some syntax issue.  The other tables all render so I don't know what the different is.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 5]
```diff
@@ -250,6 +251,20 @@ spec:
...
+| Field | Type | Description | Required
+| --- | --- | --- | --- | --- |
+| `policyName` | string | Name of the Index State Management policy. | Yes |
+| `indexPattern` | string | Index pattern to which the policy will attach. This can be an index name or pattern like `my-index-*`. | Yes |
```
Comment (kminder) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:258]: Wondering if apply is better than attach.  Do the OpenSearch documents use attach?

Comment (anders-swanson) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md:258]: They say match, like 
> ...when an index is created that matches the index template pattern, the index will have the policy attached to it

Comment (anders-swanson) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md:258]: So maybe we should say something like: 
> The index pattern is a matcher for indices to which the policy will attach

Comment (gilbode) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md:258]: If there is wording in the OpenSearch documentation perhaps we should try to be consistent with however they say things.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 6]
```diff
@@ -250,6 +251,20 @@ spec:
...
+| `minIndexAge` | [Time](https://opensearch.org/docs/1.3/opensearch/units/) | Amount of time until a managed index is deleted. Default is 7 days (7d). | No |
+| `rollover` | [Rollover](#opensearch-index-management-rollover) | Index rollover settings. | No |
+
+#### OpenSearch Index Management Rollover
+| Field | Type | Description | Required
+| --- | --- | --- | --- | --- |
+| `minIndexAge` | [Time](https://opensearch.org/docs/1.3/opensearch/units/) | Amount of time until a managed index is rolled over. Default is 1 day (1d). | No |
```
Comment (gilbode) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:265]: These URL links are to the 1.3 docs, we're using 1.2.  I think you should create a shortcode or hugo parameter or something so when we rev opensearch we don't have to go searching for all the URL refs to change them.

**Review by kminder — APPROVED** [review]
Approved but please check the rendered version before publishing.  The policy tables still aren't rendering well here.
https://github.com/verrazzano/docs/blob/76e7b3f5dd0927068ce796ec59108f811ae4deee/content/en/docs/reference/API/Verrazzano/Verrazzano.md#opensearch-index-management-policies

**Pull request merged**